demo/prepare-devnet.sh

```diff
@@ -9,7 +9,7 @@ TARGETDIR="devnet"
 
 [ -d "$TARGETDIR" ] && { echo "Cleaning up directory $TARGETDIR" ; sudo rm -r $TARGETDIR ; }
 
-cp -afT "$BASEDIR/hydra-cluster/config" "$TARGETDIR"
+cp -af "$BASEDIR/hydra-cluster/config/." "$TARGETDIR"
```

Comment:
Question: Why adding `/.` at end of path?

Reply:
It's a bit of a "trick". If you do something like `cp -r /a/.../z dir` then cp will create a directory called `z` in `dir` (and copy recursively in there). The `.` in paths refers to the directory you are in, so if you do `cp -r /a/.../z/. dir` cp will create `dir/.` and copy everything in there, but of course `dir/.` is just `dir` so this achieves the same as `-T` (this works fine for directories but have some caveats if cp -T is targeting a file).

Found this from [this](https://unix.stackexchange.com/questions/288774/what-is-the-added-value-of-the-t-option-in-gnu-cp-and-mv) SE discussion.

Having said this, I realised as typing this that in this case we don't even need the final `.` as adding a `/` at the end of config should have the same semantics (copy contents of the folder into target)